File: UVa/400/408.py
from fractions import *

while True:
    try:
        n,m = map(int,input().split())
    except EOFError:
        exit()
        
    if gcd(n, m) == 1:
        print("%10d%10d    Good Choice\n" % (n, m));
    else:
        print("%10d%10d    Bad Choice\n" % (n, m));
    

File: UVa/400/408.cpp
#include <bits/stdc++.h>
#include "header.h"

int main() {
	int step, mod;
	while (cin >> step >> mod)
		print("%10d%10d    %s Choice\n", step, mod, gcd(step, mod) == 1 ? "Good" : "Bad");
}
